netbox/utilities/api.py

```diff
@@ -95,23 +97,37 @@ def get_prefetches_for_serializer(serializer_class, fields_to_include=None):
         if serializer_field and serializer_field.source:
             model_field_name = serializer_field.source
 
+        # If the serializer field is a RelatedObjectCountField and its a nested field
+        # Add an annotation to the annotaded_prefetch
+        if isinstance(serializer_field, RelatedObjectCountField) and source_field is not None:
+            if model_field_name not in annotaded_prefetch:
+                annotaded_prefetch[model_field_name] = Count(serializer_field.relation)
```

Comment:
Do we need to continue processing at this point?  I think we can just do a continue here?